pkg/pillar/hypervisor/kvm.go

```diff
@@ -626,6 +626,31 @@ func (ctx kvmContext) GetDomsCPUMem() (map[string]types.DomainMetric, error) {
 	}
 
 	for dom, pid := range ctx.domains {
+		//Fetch cpu usage from top output
+		//FIXME: Once we migrate to kvmtop util, remove this logic
```

Comment:
Let me file a tracker story and assign it to you, ok @cshari-zededa ?

Reply:
> No, I don't want more screen scraping of some top command output. Far too brittle.
> gopsutil seems reasonable. Let's do that.

Done. and updated the PR.